input/pagecontent/concepts_onboarding_checklist.md

```diff
@@ -130,13 +131,18 @@ This repository contains the template for building [onboarding](https://github.c
 > Note: OSSL_COUNTRY_NAME should be ISO 2 letter name of the country mapped to the name used in repository.
 	
 - Then execute the script. It will generate all certificates and keys in a subfolder named by current datetime.
+- While execution of the "gen_all_certs.sh" script, Please provide script agrument "DN_template.cnf" file  
+  which conisit of county related info to generate all required certificates (TLS,SCA,UP)
```

Comment:
spell check for "consists" country or country? 

info -> information 

@arajnor99

Reply:
Correction done .
https://github.com/WorldHealthOrganization/smart-trust/blob/42328c7cdef638362ab83bb2e2952f23bd918e66/input/pagecontent/concepts_onboarding_checklist.md?plain=1#L135